src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -106,6 +106,10 @@ public boolean compile(String cp) throws IOException, InterruptedException {
         useGlisten = GlistenResolver.useGlisten(cp);
         String nativeImage = getNativeImagePath();
         ProcessBuilder compileBuilder = new ProcessBuilder(nativeImage);
+        List<String> buildTimeList = getInitializeAtBuildTimeList(useGlisten);
```

Comment:
This is one for @eugener : do we want to have this boolean inside the method, or on the configuration?

Reply:
Agree, we'll need a better way to handle the different "uses", but I'd say we can do that in a follow up PR?

Reply:
Agree with @jperedadnr... we should try to redesign the resolvers using "strategy" pattern. But that can be done in a later PR - no sense of holding this one.